docs/configuration-reference/components/aws-ebs-csi-driver.md

```diff
@@ -53,14 +54,15 @@ component "aws-ebs-csi-driver" {
+| `enable_volume_resizing`       | Expand the volume size after the initial provisioning.                                                                                                                                                                                                                                                                      | `true`                                | bool                                                                                                             | false    |
+| `enable_volume_snapshot`       | Create Volume snapshots from an existing EBS volume for backup and restore.                                                                                                                                                                                                                                                 | `true`                                | bool                                                                                                             | false    |
+| `node_affinity`                | Node affinity for deploying the EBS CSI controller and EBS Snapshot controller pods.                                                                                                                                                                                                                                        | -                                     | `list(object({key = string, operator = string, values = list(string)}))`                                         | false    |
+| `tolerations`                  | Tolerations that the EBS CSI Node, EBS CSI controller and EBS Snapshot controller pods will tolerate.                                                                                                                                                                                                                       | `tolerations { operator = "Exists" }` | `list(object({key = string, effect = string, operator = string, value = string, toleration_seconds = string }))` | false    |
+| `reclaim_policy`               | The persistent volumes created with this storage class will have this reclaim policy. This field decides what happens to the volume after a user deletes a PVC. Valid values: `Retain`, `Recycle` and `Delete`. Read more in [Kubernetes docs](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#reclaiming). | `Retain`                              | string                                                                                                           | false    |
```

Comment:
How about this: `Persistent volumes created with this storage class will have reclaim policy mentioned in this field.` ?

Isn't it understood that a administrator creates a SC which users use to creates PVs? Or should we explain that?

Reply:
Looking a bit more into this, I think this parameter means different things in different components:

* aws-ebs-csi-driver creates one storage class, so we can say something like "Persistent volumes created with the storage class created by this component will have this reclaim policy...".
* In openebs-storage-class the parameter is defined inside the `storage-class` block so the phrasing "Persistent volumes created with this storage class will have this reclaim policy..." makes sense.
* In rook-ceph there's a `storage_class` block that can be enabled or not, since the `reclaim_policy` field is inside this block, the phrasing "Persistent volumes created with this storage class will have this reclaim policy..." is fine too.